### DISCUSSION TITLE
Where does the requirement for an item name come from?

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#52] [General]
Where does the requirement for an item's name come from?  For example, the in-network object has a field called "name" and that is specified as required.  I can't find any requirement for this in the rules.  

In contrast, there are several mentions of "a plain language description for each billing code" and it is fairly clear that it is required.

**Comment by taylorpatriciab** [comment]
Are standard descriptions required or is it free form text to be provided by each submitter

**Reply by bryanbriegel** [reply]
@taylorpatriciab  - the ICR indicates "Brief description of the item or service. In the case of items and services that are associated with common billing codes (such as the HCPCS codes), the codes’ associated short text description may be provided. In the case of NDCs for prescription drugs, the plain language description must be the proprietary and nonproprietary names assigned to the NDC by the FDA" - appears 'optional re; may' for common billing codes, mandated text for NDC codes.

**Reply by ghost** [reply]
That is how we have view it and the text we will be providing for the "name" field in the schema in In-Network we will be using the short text.

**Reply by nmayle** [reply]
It doesn't seem like my question was answered.  The In-Network File description has two separate fields for textual description of the item, one for the In-Network Object name and one for the In-Network Object description.  The rule doesn't seem to require two separate fields, so why have two separate fields?  The name field doesn't seem to have **any** support in the rule itself.  Can one of the authors please cite where the rule requires this.

**Reply by reevesj2** [reply]
@nmayle did you ever receive any type of answer?  I am not following the use of the two fields either.

**Reply by tdfow** [reply]
@reevesj2 @nmayle currently, we are populating the name/description with same content as we don't have a separate field for 'name'.
